tests/test_astrometryTask.py

```diff
@@ -112,6 +112,37 @@ def testUsedFlag(self):
                 count += 1
         self.assertEqual(count, len(results.matches))
 
+    def testMaxMeanOffset(self):
+        """If the astrometric fit does not satisfy the maxMeanOffsetArcsec
+        threshold, ensure task raises a RuntimeError.
+        """
+        self.exposure.setWcs(self.tanWcs)
+        loadRes = self.refObjLoader.loadPixelBox(bbox=self.bbox, wcs=self.tanWcs, filterName="r")
+        refCat = loadRes.refCat
+        refCentroidKey = afwTable.Point2DKey(refCat.schema["centroid"])
+        refFluxRKey = refCat.schema["r_flux"].asKey()
+
+        sourceSchema = afwTable.SourceTable.makeMinimalSchema()
+        measBase.SingleFrameMeasurementTask(schema=sourceSchema)  # expand the schema
+        config = AstrometryTask.ConfigClass()
+        config.maxMeanOffsetArcsec = 0.0  # To ensure a "deemed" WCS failure
+        # schema must be passed to the solver task constructor
+        solver = AstrometryTask(config=config, refObjLoader=self.refObjLoader, schema=sourceSchema)
+        sourceCat = afwTable.SourceCatalog(sourceSchema)
+        sourceCat.reserve(len(refCat))
+        sourceCentroidKey = afwTable.Point2DKey(sourceSchema["slot_Centroid"])
+        sourceInstFluxKey = sourceSchema["slot_ApFlux_instFlux"].asKey()
+        sourceInstFluxErrKey = sourceSchema["slot_ApFlux_instFluxErr"].asKey()
+
+        for refObj in refCat:
```

Comment:
Why not just use the existing `makeSourceCat` method in this test? I think that could replace a bunch of your catalog management in this test.

Reply:
Thanks for pointing that out!  I guess I very lazily just copy-pasta'd from the `testUsedFlag` test above and didn't pay attention to that method.  I've actually updated it for ease of reuse and to add some optional extra functionality.  I updated both my new test and `testUsedFlag`  above to use that function.